Traceback (most recent call last):
  File "/usr/bin/gcovr", line 33, in <module>
    sys.exit(load_entry_point('gcovr==5.0', 'console_scripts', 'gcovr')())
  File "/usr/lib/python3/dist-packages/gcovr/__main__.py", line 280, in main
    error_occurred = print_reports(covdata, options, logger)
  File "/usr/lib/python3/dist-packages/gcovr/__main__.py", line 432, in print_reports
    if generator(covdata, output.abspath, options):
  File "/usr/lib/python3/dist-packages/gcovr/writer/html.py", line 401, in print_html_report
    lines = formatter.highlighter_for_file(data['filename'])(source_file.read())
  File "/usr/lib/python3/dist-packages/gcovr/writer/html.py", line 159, in highlighter_for_file
    from jinja2 import Markup
ImportError: cannot import name 'Markup' from 'jinja2' (/home/subramd/.local/lib/python3.10/site-packages/jinja2/__init__.py)
